code/modules/reqs/supply.dm

```diff
@@ -636,6 +636,7 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 /obj/item/storage/backpack/marine/radiopack/Initialize(mapload, ...)
 	. = ..()
 	AddComponent(/datum/component/beacon, FALSE, 0, icon_state + "_active")
+	START_PROCESSING(SSobj, src)
```

Comment:
Bad idea, just register a signal to COMSIG_MOVABLE_MOVED on activate so that the position follows you instead of processing constantly from the moment it's created.

Reply:
Thanks, I'll look into that